src/main/java/frc/robot/subsystems/elevator/ElevatorIOSim.java

```diff
@@ -39,7 +39,7 @@ public void updateInputs(final ElevatorIOInputsAutoLogged inputs) {
     inputs.elevatorPositionMeters = physicsSim.getPositionMeters();
     inputs.elevatorVelocityMetersPerSec = physicsSim.getVelocityMetersPerSecond();
     inputs.elevatorAppliedVolts = volts;
-    inputs.elevatorCurrentAmps = new double[] {physicsSim.getCurrentDrawAmps()};
+    inputs.elevatorStatorCurrentAmps = new double[] {physicsSim.getCurrentDrawAmps()};
```

Comment:
sometimes using arrays can be confusing the total length. also since this length can be whole replaced by a new array (which looks like is what you are doing everytime -- constructing a new array instead of setting existing values).